batch7crud-roll5/src/main/webapp/css/layout.css

```diff
@@ -0,0 +1,58 @@
+/* header */
+.header-wrapper {
+	width: 100%;
+	background-color: #323232;
+	padding-top: 30px;
+	padding-bottom: 30px;
+	line-height: 0px;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unit for this zero value [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=css%3Azero-units)